jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_ECEN

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+source "${HOMEgfs}/ush/preamble.sh"
+export WIPE_DATA="NO"
+export DATA="${DATAROOT}/${RUN}ocnanal_${cyc}"
```

Comment:
Is this job supposed to reuse this DATA directory?

Reply:
No, that's a left over copy/paste, needs to be corrected @AndrewEichmann-NOAA .

Reply:
fixed